### ISSUE TITLE
更新后加载失败module 'execution' has no attribute 'recursive_execute'

### ISSUE THREAD

**Issue by 807502278** [ty0x2333/ComfyUI-Dev-Utils#15]
comfyui和ComfyUI-Dev-Utils都更新了(2024-8-16)
启动提示：
```
Warn!: Traceback (most recent call last):
  File "D:\AI\ComfyUI-3D\ComfyUI\nodes.py", line 1982, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "D:\AI\ComfyUI-3D\ComfyUI\custom_nodes\ComfyUI-Dev-Utils\__init__.py", line 3, in <module>
    from .nodes.execution_time import ExecutionTime
  File "D:\AI\ComfyUI-3D\ComfyUI\custom_nodes\ComfyUI-Dev-Utils\nodes\execution_time.py", line 24, in <module>
    origin_recursive_execute = execution.recursive_execute
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'execution' has no attribute 'recursive_execute'

Warn!: Cannot import D:\AI\ComfyUI-3D\ComfyUI\custom_nodes\ComfyUI-Dev-Utils module for custom nodes: module 'execution' has no attribute 'recursive_execute'
```

**Comment by liushh39** [comment]
解决了吗

**Comment by wudideasdf** [comment]
same error

**Comment by whmc76** [comment]
+1

**Comment by Willy-Shenn** [comment]
@hugovntr has already submitted the modified code, but the author has not yet merged it into the existing program for an update. You just need to replace \\custom_nodes\ComfyUI-Dev-Utils\nodes\execution_time.py with the following file. I hope it works successfully. Thanks again to the author and to @hugovntr for submitting this update. Have a great day!
[execution_time.zip](https://github.com/user-attachments/files/16640214/execution_time.zip)
![image](https://github.com/user-attachments/assets/b39c1d35-da0b-46d0-8e06-aa9e57fa9fc4)

**Comment by ty0x2333** [comment]
Thank you.

PR https://github.com/ty0x2333/ComfyUI-Dev-Utils/pull/16 by @hugovntr has fixed this bug.

I have now released ComfyUI-Dev-Utils 0.4.3. The issue has been resolved in the latest version, which supports both the older and the latest versions of ComfyUI.

Please upgrade ComfyUI-Dev-Utils to the latest version.

@807502278 @wudideasdf @liushh39 @whmc76  @Willy-Shenn
